lib/lib.go

```diff
@@ -0,0 +1,7 @@
+package lib
+
+import (
+	"github.com/sirupsen/logrus"
+)
+
+var log = logrus.New()
```

Comment:
It might be nice to have some way of configuring the level of this logger via an argument or environment variable. It took me a stupidly long time to realize I needed to add an `init()` func and set it here instead of trying to do it in `root.go` (d'oh, different packages) :P